### ISSUE TITLE
Incremental backups strategy

### ISSUE THREAD

**Issue by tms-epcc** [wfau/ScienceArchives#542]
- ROE generated data products (ie category 3 in the backup strategy document)
incremental back-up takes ages due to the large volumes -- so trying to come up with a smarter strategy.  
    - Rob will speak to Eckhard about a two part strategy for this based on active and inactive usage.
        - Discussed with Eckhard.
        - The config file has a list of disks that are no longer being changed.  The plan is to treat these as static data and backup once per year so only need incremental backup for more recent dynamic data.
        - Next will check if incremental back ups for this more recent dynamic data are feasible.  
          - Little progress since Eckhard and Rob not available at same time. 
- Once plan agreed, documentation on this will need to be added to the existing documentation.

**Comment by tms-epcc** [comment]
19/JUL/24
- see #547 re the card

**Comment by tms-epcc** [comment]
13/SEP/24
- card now fixed so this will now be the main priority

**Comment by tms-epcc** [comment]
12/DEC/24
- Progress with check if incremental back ups for this more recent dynamic data are feasible?

**Comment by tms-epcc** [comment]
24/JAN/25
- documentation to be updated while implementing 
- will experiment first if can update incrementals while baciking up the DBs

**Comment by tms-epcc** [comment]
28/MAR/25
- now have new tapes in C2 can progress this

**Comment by tms-epcc** [comment]
25/APR/25
- still try new tapes

**Comment by tms-epcc** [comment]
23/MAY/25
- some tests run but problems arose in allocating tapes for backup strategies
- need to resolve configuration

**Comment by tms-epcc** [comment]
19/JUN/25
- expect to finish DB backups by end July and can then work on incremental

**Comment by tms-epcc** [comment]
17/JUL/25
- once DB tapes in fire safe will start on this 
- need to confirm number of new tapes needed

**Comment by tms-epcc** [comment]
17/SEP/25
- after speaking to MarkH, need many new tapes (2PB worth) - have a price for tapes - waiting for Bob to decide
- Bob asked to ask if RAL can do this instead but waiting for reply and price

**Comment by tms-epcc** [comment]
28/OCT/25
- still waiting for RAL, will ask Bob

**Comment by tms-epcc** [comment]
03/DEC/25
- see #544

**Comment by tms-epcc** [comment]
17/JUL/26
- this needs to be reviewed once the migration of the backup to RAL is complete

**Comment by tms-epcc** [comment]
28/AUG/26
- superseded by RAL so not necessary
